src/ua/shalypenko/JDBC/JDBCMethods.java

```diff
@@ -0,0 +1,155 @@
+package ua.shalypenko.JDBC;
+
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.Scanner;
+
+public class JDBCMethods {
+
+    public static String input(String message) {
+        Scanner scan = new Scanner(System.in);
+        System.out.println(message);
+        return scan.nextLine();
+    }
+
+    public static void insertIntoTableContact() {
+        try (Connection connection = ConnectionManager.getConnection()) {
+            String query = "INSERT INTO Alevelgraduates.contact(contact, contact_type) VALUES (?,?)";
+            PreparedStatement preparedStatement = connection.prepareStatement(query);
+            System.out.println("Next you will add data to the table 'contact'");
+            preparedStatement.setString(1, input("Enter new contact"));
+            preparedStatement.setString(2, input("Enter new contact_type"));
+            int result = preparedStatement.executeUpdate();
+            System.out.println(result);
+        } catch (SQLException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    public static void createNewPersonInTableEmployee() {
+        try (Connection connection = ConnectionManager.getConnection()) {
+            String query = "INSERT INTO Alevelgraduates.employee(first_name, lust_name) VALUES (?,?)";
+            PreparedStatement preparedStatement = connection.prepareStatement(query);
+            System.out.println("Next you will add data to the table 'employee'");
+            preparedStatement.setString(1, input("Enter first name"));
+            preparedStatement.setString(2, input("enter last name"));
+            int result = preparedStatement.executeUpdate();
+            System.out.println(result);
+        } catch (SQLException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    public static void selectFromTableEmployee() {
+        try (Connection connection = ConnectionManager.getConnection()) {
+            String graduates = "SELECT * from Alevelgraduates.employee";
+            PreparedStatement preparedStatement = connection.prepareStatement(graduates);
+            ResultSet graduatestSet = preparedStatement.executeQuery();
+            while (graduatestSet.next()) {
+                String lustName = graduatestSet.getString("lust_name");
+                String firstName = graduatestSet.getString("first_name");
+                String contact = graduatestSet.getString("contact_id");
+                System.out.println(firstName + " " + lustName + " has contact_id # " + contact);

```

Comment:
When you executing your statements, you just print field to console without mapping relational model to object. Do not forget, that our DAO is designed to do this mapping from string values from DB to java objects.

For example, here when you select
<img width="498" alt="image" src="https://github.com/DanyloShalypenko/A-LevelHW/assets/72043323/abe91991-1c56-418f-ad21-13611ecac025">


Instead of just System.out.println you need to create object, add to some list and return this list (if this select of all object).

If you have any questions regarding it, pm.

Other minor note, dao methods should be unified(namings and arguments). But it's more for the next homework, so, don't mind.